SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -261,6 +261,14 @@ class SPATIALGDKEDITOR_API USpatialGDKEditorSettings : public UObject
 	FFilePath SpatialOSLaunchConfig;
 
 public:
+	/** Expose the runtime on a particular IP address when it is running on this machine. */
+	UPROPERTY(EditAnywhere, config, Category = "Launch", meta = (ConfigRestartRequired = false, DisplayName = "Expose local runtime"))
+	bool bExposeRuntimeIP;
```

Comment:
I've noticed there are multiple `if` statements that branch on `bExposeRuntimeIP` variable. However, there are multiple checks for `ExposedRuntimeIP` being empty in `FLocalDeploymentManager`. So maybe we don't need `bExposeRuntimeIP`, or at least we can remove checks `if(bExposeRuntimeIP)`?

Reply:
We discussed this offline, but for the sake of documentation I'm adding the reasoning here.

I think that keeping the `bExposeRuntimeIP` would be more user friendly. The settings entries look as follows right now:

![image](https://user-images.githubusercontent.com/54983025/65593565-a77c5380-df88-11e9-89cc-518a3d9efb15.png)

This allows quick toggling between (non)exposure, and remembers the IP field. The excessive branching seems to me like a necessary evil for this user comfort.